src/main/java/org/opentripplanner/routing/impl/DefaultFareServiceFactory.java

```diff
@@ -163,7 +163,9 @@ public static FareServiceFactory fromConfig(JsonNode config) {
             retval = new OrcaFareServiceFactory();
             break;
         case "pdx":
-            retval = new PdxFareServiceFactory();
+            LOG.warn("The `pdx` fare factory is deprecated. Please switch to `highestFareInFreeTransferWindow`.");
```

Comment:
It might be simpler to just keep the pdx fare factory and have it utilize highestFareInFreeTransferWindow under the hood with pdx default values for transfer window and interlined stuff.

Reply:
Documentation added in 666783e02461c0db864b156c70230b4c2812eefa.